<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFlight</class>
 <widget class="QWidget" name="AddFlight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1258</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="styleSheet">
      <string notr="true">background-color: #f0f0f0;
border: 1px solid #ccc;
border-radius: 5px;
background-repeat:no-repeat;
background-position:center;
background-image: url(:/ing/AddFlightImg.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="minimumSize">
         <size>
          <width>265</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>12000000</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#label_8 {
background:none;
    background-color: #f7f7f7; /* Light grey background */
    color: #333333; /* Dark grey text */
    border: 1px solid #d9d9d9; /* Light grey border */
    border-radius: 5px; /* Rounded corners */
    padding: 5px 10px; /* Padding around the text */
    font-size: 20px; /* Font size appropriate for the label size */
    font-weight: bold; /* Bold text */
    text-align: center; /* Center the text */
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); /* Subtle shadow effect */
    margin-bottom: 10px; /* Space below the label */
    letter-spacing: 1px; /* Spacing between letters */
    height: 30px; /* Explicit height for the label */
}

</string>
        </property>
        <property name="text">
         <string>Add Flight</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frame_4">
        <property name="maximumSize">
         <size>
          <width>1143</width>
          <height>525</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:none;background-color: rgb(135, 206, 235);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QFrame" name="frame_2">
           <property name="styleSheet">
            <string notr="true">border:none;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="5">
          <widget class="QFrame" name="frame">
           <property name="minimumSize">
            <size>
             <width>1119</width>
             <height>464</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>1119</width>
             <height>464</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame{
background:none;
background-color: rgb(255, 255, 255);
}

QLable{
color: rgb(0, 0, 0);font: 10pt &quot;MS Shell Dlg 2&quot;;
}
</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>40</y>
              <width>100</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;color: rgb(0, 0, 0);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
            <property name="text">
             <string>Flight ID</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="FA_FlightID">
            <property name="geometry">
             <rect>
              <x>190</x>
              <y>40</y>
              <width>150</width>
              <height>30</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #f7f7f7;
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 1px;
    margin-top: 1px;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); 
color: rgb(0, 0, 0);</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>110</y>
              <width>100</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;color: rgb(0, 0, 0);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
            <property name="text">
             <string>Flight Name</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>190</y>
              <width>100</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;color: rgb(0, 0, 0);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
            <property name="text">
             <string>Arrival</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>260</y>
              <width>100</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;color: rgb(0, 0, 0);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
            <property name="text">
             <string>Departure</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>330</y>
              <width>100</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;color: rgb(0, 0, 0);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
            <property name="text">
             <string>Duration</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="FA_FlightName">
            <property name="geometry">
             <rect>
              <x>190</x>
              <y>110</y>
              <width>200</width>
              <height>30</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #f7f7f7;
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 1px;
    margin-top: 1px;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); 
color: rgb(0, 0, 0);</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="FA_Duration">
            <property name="geometry">
             <rect>
              <x>190</x>
              <y>330</y>
              <width>200</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #f7f7f7;
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 1px;
    margin-top: 1px;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); 
color: rgb(0, 0, 0);</string>
            </property>
           </widget>
           <widget class="QComboBox" name="FA_Arrival">
            <property name="geometry">
             <rect>
              <x>190</x>
              <y>190</y>
              <width>200</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background:none;</string>
            </property>
            <item>
             <property name="text">
              <string>Delhi</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Kolkata</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Channai</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mumbai</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Benglor</string>
             </property>
            </item>
           </widget>
           <widget class="QComboBox" name="FA_Departure">
            <property name="geometry">
             <rect>
              <x>190</x>
              <y>260</y>
              <width>200</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background:none;</string>
            </property>
            <item>
             <property name="text">
              <string>Delhi</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Kolkata</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Channai</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mumbai</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Benglor</string>
             </property>
            </item>
           </widget>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>590</x>
              <y>30</y>
              <width>100</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>Class</string>
            </property>
           </widget>
           <widget class="QDateEdit" name="FA_DateOfFlight">
            <property name="geometry">
             <rect>
              <x>750</x>
              <y>250</y>
              <width>150</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QDateEdit {
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 5px 10px;
    background-color: #ffffff;
    font-size: 14px;
    color: #444;
    width: 150px;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); /* Light shadow for a floating effect */
}

QDateEdit::hover {
    border: 1px solid #bbb;
    background-color: #f7f7f7f7;
}

QDateEdit::focus {
    border: 1px solid #4CAF50;
    background-color: #e0f7e7;
    box-shadow: 0 2px 8px rgba(0, 0, 0, 0.2); /* Stronger shadow on focus */
}

QDateEdit::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left: 1px solid #ddd;
    background-color: #f0f0f0;
    border-top-right-radius: 6px;
    border-bottom-right-radius: 6px;
    padding-right: 5px;
}

QDateEdit::down-arrow {
    width: 12px;
    height: 12px;
}

QDateEdit::down-arrow:hover {
}

QDateEdit::up-arrow, QDateEdit::down-arrow {
    width: 12px;
    height: 12px;
}

QDateEdit::up-button, QDateEdit::down-button {
    subcontrol-origin: border;
    subcontrol-position: center right;
    width: 16px;
    height: 16px;
    background-color: transparent;
}

QDateEdit::drop-down:pressed {
    background-color: #d6d6d6;
}

QDateEdit::drop-down:focus {
    border-left: 1px solid #4CAF50;
}

QDateEdit QAbstractItemView {
    border: 1px solid #ccc;
    border-radius: 4px;
    background-color: #f7f7f7;
    selection-background-color: #4CAF50;
    selection-color: #fff;
}</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_11">
            <property name="geometry">
             <rect>
              <x>590</x>
              <y>250</y>
              <width>120</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;color: rgb(0, 0, 0);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
            <property name="text">
             <string>Date of Flight</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_10">
            <property name="geometry">
             <rect>
              <x>590</x>
              <y>180</y>
              <width>100</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;color: rgb(0, 0, 0);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
            <property name="text">
             <string>Fare</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_9">
            <property name="geometry">
             <rect>
              <x>590</x>
              <y>100</y>
              <width>100</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;color: rgb(0, 0, 0);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
            <property name="text">
             <string>Seats</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_12">
            <property name="geometry">
             <rect>
              <x>590</x>
              <y>320</y>
              <width>120</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;color: rgb(0, 0, 0);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
            </property>
            <property name="text">
             <string>Time of Flight</string>
            </property>
           </widget>
           <widget class="QTimeEdit" name="FA_TimeOfFlight">
            <property name="geometry">
             <rect>
              <x>750</x>
              <y>320</y>
              <width>150</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QTimeEdit {
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 5px 10px;
    background-color: #ffffff;
    font-size: 14px;
    color: #444;
    width: 150px;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); /* Light shadow for a floating effect */
}

QTimeEdit::hover {
    border: 1px solid #bbb;
    background-color: #f7f7f7f7;
}

QTimeEdit::focus {
    border: 1px solid #4CAF50;
    background-color: #e0f7e7;
    box-shadow: 0 2px 8px rgba(0, 0, 0, 0.2); /* Stronger shadow on focus */
}

QTimeEdit::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left: 1px solid #ddd;
    background-color: #f0f0f0;
    border-top-right-radius: 6px;
    border-bottom-right-radius: 6px;
    padding-right: 5px;
}

QTimeEdit::down-arrow {
    width: 12px;
    height: 12px;
}

QTimeEdit::down-arrow:hover {
}

QTimeEdit::up-arrow, QDateEdit::down-arrow {
    width: 12px;
    height: 12px;
}

QTimeEdit::up-button, QTimeEdit::down-button {
    subcontrol-origin: border;
    subcontrol-position: center right;
    width: 16px;
    height: 16px;
    background-color: transparent;
}

QTimeEdit::drop-down:pressed {
    background-color: #d6d6d6;
}

QTimeEdit::drop-down:focus {
    border-left: 1px solid #4CAF50;
}

QTimeEdit QAbstractItemView {
    border: 1px solid #ccc;
    border-radius: 4px;
    background-color: #f7f7f7;
    selection-background-color: #4CAF50;
    selection-color: #fff;
}</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="FA_Fare">
            <property name="geometry">
             <rect>
              <x>750</x>
              <y>180</y>
              <width>200</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #f7f7f7;
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 1px;
    margin-top: 1px;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); 
color: rgb(0, 0, 0);</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>920</x>
              <y>260</y>
              <width>111</width>
              <height>25</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>(MM/DD/YYYY)</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="FA_Seat">
            <property name="geometry">
             <rect>
              <x>750</x>
              <y>100</y>
              <width>200</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #f7f7f7;
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 1px;
    margin-top: 1px;
    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); 
color: rgb(0, 0, 0);</string>
            </property>
           </widget>
           <widget class="QComboBox" name="FA_Class">
            <property name="geometry">
             <rect>
              <x>750</x>
              <y>30</y>
              <width>200</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background:none;</string>
            </property>
            <item>
             <property name="text">
              <string>Economy</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Business</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>First Class</string>
             </property>
            </item>
           </widget>
           <widget class="QCalendarWidget" name="calendarWidget">
            <property name="geometry">
             <rect>
              <x>750</x>
              <y>283</y>
              <width>291</width>
              <height>181</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QCalendarWidget {
    font: 5pt &quot;Segoe UI&quot;;
    background-color: #f0f0f0;
    border: 1px solid #ccc;
    border-radius: 5px;
}

QCalendarWidget QWidget {
    background-color: #f0f0f0;
}

QCalendarWidget QToolButton {
    background-color: #1E90FF;
    color: white;
    border: none;
    border-radius: 5px;
    padding: 3px;
    font-size: 6pt; /* Larger font size for the month and year */
}

QCalendarWidget QToolButton::menu-indicator {
    image: none;
}

QCalendarWidget QMenu {
    width: 150px;
    background-color: white;
    border: 1px solid #ccc;
}

QCalendarWidget QSpinBox {
    margin: 2px;
    font-size: 6pt; /* Match the font size with the QToolButton */
}

QCalendarWidget QSpinBox::up-button {
    subcontrol-origin: border;
    subcontrol-position: top right;
    width: 15px;
}

QCalendarWidget QSpinBox::down-button {
    subcontrol-origin: border;
    subcontrol-position: bottom right;
    width: 15px;
}

QCalendarWidget QAbstractItemView:enabled {
    font-size: 5pt; /* Keeping dates at 5pt */
    color: #000000;
    background-color: #f0f0f0;
    selection-background-color: #1E90FF;
    selection-color: white;
}

QCalendarWidget QAbstractItemView:disabled {
    color: #999999;
}

QCalendarWidget QHeaderView::section {
    background-color: #1E90FF;
    color: white;
    font-weight: bold;
    font-size: 6pt; /* Larger font size for the headers */
}
</string>
            </property>
           </widget>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QFrame" name="frame_5">
           <property name="styleSheet">
            <string notr="true">border:none;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="Cancel">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background:none;
	background-color: rgb(255, 255, 255);
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}
</string>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="FA_Add">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background:none;
	background-color: rgb(255, 255, 255);
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}</string>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
